apps/public-docsite-v9/src/Concepts/BrowserSupportMatrix.stories.mdx

```diff
@@ -19,9 +26,54 @@ FluentUI supports browsers that are at or above the indicated versions below:
 | >=14.5        | >=84               | >=16    |
 
 FluentUI will ensure that no browser, EcmaScript(ES), or CSS features that are incompatible with the support matrix laid out above are introduced.
-FluentUI will not be providing polyfills for features that we expect our browser support matrix to already support. This means that consumers who require the
-use/support of older browsers that don't support certain browser, ES, or CSS features (i.e. `ResizeObserver`),
-are expected to implement their own polyfills or use a third-party one.
+FluentUI will not be providing our own polyfills for features that we expect our browser support matrix to already support.
+There are often well-tested third-party polyfills available that applications can use to fill in the gaps.
+Consumers who require the use/support of older browsers that don't support certain browser, ES, or CSS features (i.e. ResizeObserver),
+will need to implement or use a third-party polyfill to provide the functionality. See the partial support section below for features
+that may need to be polyfilled.
+
+## Partial Browser Support Matrix:
+
+FluentUI provides partial support for the browsers below which will have limitations that will require consumers to transpile down and/or use polyfills to cover the support gaps.
+
+**Desktop Browsers:**
+
+| Edge | Firefox | Chrome | Safari | Opera | Internet Explorer |
+| ---- | ------- | ------ | ------ | ----- | ----------------- |
+| >=79 | >=69    | >=79   | >=13.1 | >=64  | Not Supported     |
+
+**Mobile Browsers:**
+
+| Safari on iOS | Chrome for Android | Samsung |
+| ------------- | ------------------ | ------- |
+| >=13.4        | >=79               | >=14    |
+
+List of features that are currently used in FluentUI that aren't supported out of the box by the partial support matrix above:
+
+**Unsupported CSS Features:**
+
+1.  Flex gap: Unsupported by Chrome, Edge, Firefox, Opera, Safari, Safari on IOS, Chrome on Android
+2.  CSS min, max, clamp: Unsupported by Firefox and Opera
+3.  CSS revert value: Unsupported by Firefox and Opera
+
+**Unsupported ECMAScript Features:**
+As outlined in the very beginning, FluentUI will be using ES2020 as its Typescript target; so ES2020 features such as nullish coalescing and optional chaining are already being utilized by the library.
+These ES2020 features are not guaranteed to work in all browsers within the partial support matrix so consumers will need to perform down-level transpilation of the library.
```

Comment:
Is there a relation or intersection between partial browser support and ECMAScript? I feel like our ECMAScript support is orthogonal to the browser support matrix since we expect apps to transpile to whatever specific browser they need to.

If that's accurate, I would pull this out into the ECMAScript section

Reply:
I condensed this section a bit but decided to keep it to make it clear that in the case where a consumer wants to use the ES2020 bundle we ship and for some reason decides to follow the partial support matrix that they know it won't work.